src/main/scala/cromwell/engine/ExecutionTable.scala

```diff
@@ -0,0 +1,11 @@
+package cromwell.engine
+
+import cromwell.binding.{Call, WdlBinding}
+
+/**
+ * Corresponds to the "execution table" of our discussions.
+ */
```

Comment:
It took me a while to realize what bothered me about these names but I dislike ExecutionTable and SymbolTable. I realize why they're named the way they are but they're collections of objects and there's no guarantee they'll ever be stored/persisted in any rectangular form in the future. I'd prefer different names.